.travis.yml

```diff
@@ -1,7 +1,13 @@
+
+---
 language: python
-python: 3.6
+python: 2.7
 services: docker
+install:
+- git clone --recursive https://github.com/systemslab/popper /tmp/popper
```

Comment:
we might want to avoid cloning master and, instead, delete this line and use the version that we are testing. So the `export` line below would be:

```bash
export PATH=$PATH:$PWD/cli/bin
```

Reply:
yeah, I'll go ahead and do that.